carrier.go

```diff
@@ -0,0 +1,175 @@
+package goshopify
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/shopspring/decimal"
+)
+
+const carrierBasePath = "carrier_services"
+
+// CarrierServiceService is an interface for interfacing with the carrier service endpoints
+// of the Shopify API.
+// See: https://shopify.dev/docs/admin-api/rest/reference/shipping-and-fulfillment/carrierservice
+type CarrierServiceService interface {
+	List() ([]CarrierResource, error)
+	Get(int64) (*CarrierResource, error)
+	Create(CarrierResource) (*CarrierResource, error)
+	Update(CarrierResource) (*CarrierResource, error)
+	Delete(int64) error
+}
+
+// CarrierServiceOp handles communication with the product related methods of
+// the Shopify API.
+type CarrierServiceOp struct {
+	client *Client
+}
+
+// CarrierResource represents a Shopify carrier service
+type CarrierResource struct {
```

Comment:
My bad, I forgot about this one. I just looked at the shopify.dev docs again to see how they name things, can you rename this one to `CarrierService`?

Reply:
Updated!

Reply:
Thank you!